### PR TITLE
Update the "Application" CRD document to follow the template

### PR DESCRIPTION
**Pull request by tomekpapiernik**
**Description**

Changes proposed in this pull request:

- Updated "Application" CRD document 
- Updated Application Registry description in "Architecture"
- Removed a section from the ApplicationMapping CRD document (repetition)

**Related issue(s)**
#1149

### CODE REVIEW

File: docs/application-connector/docs/02-01-application-connector.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ The Connector Service:
 
 ## Application Registry
 
-The Application Registry stores all registered APIs and Event Catalog exposed by a connected external solution. The metadata of the external solution's APIs and Event catalogs is stored in the [Application](#custom-resource-application) custom resource.
-The system creates a new Kubernetes service for each registered API. Additionally, new ServiceClasses are registered in the Service Catalog.
+The Application Registry writes and reads the APIs and Event Catalog metadata of the connected external solution in the [Application](#custom-resource-application) custom resource.
```
Comment (klaudiagrz) [inline docs/application-connector/docs/02-01-application-connector.md]: I'm not sure about using `write` verb in here. Maybe `reads and saves` ?

File: docs/application-connector/docs/06-01-application.md [hunk 2]
```diff
@@ -53,6 +53,16 @@ This table lists all the possible parameters of a given resource together with t
 | **spec.services.entries.oauthUrl** |    **NO**   | Specifies the URL used to authorize with a given API. This field is required for the API entry type. Parameter provided by the Application Registry, do not edit. |
 | **spec.services.entries.credentialsSecretName** |    **NO**   | Specifies the name of the Secret which allows you to call a given API. This field is required if **spec.services.entries.oauthUrl** is specified. Parameter provided by the Application Registry, do not edit. |
 
+## Related resources and components
+
+These components use this CR:
+
+| Component   |  Description |
+|:-----------:|:-------------|
+| Application Registry | Reads and writes the APIs and Event Catalog metadata of the connected external solution to this CR. |
```
Comment (klaudiagrz) [inline docs/application-connector/docs/06-01-application.md]: Same concern as in the previous comment.

Comment (klaudiagrz) [inline docs/application-connector/docs/06-01-application.md]:
```suggestion
| Application Registry | Reads and writes the APIs and Event Catalog metadata of the external solution connected to this CR. |
```

Comment (tomekpapiernik) [reply docs/application-connector/docs/06-01-application.md]: An external solution doesn't connect to the CR, it connects to Kyma through the Application Connector.

**Pull request merged**